reddit.js

```diff
@@ -1,5 +1,41 @@
 var reddit = require("./data.json")
 
 // Write your solutions below
+// console.log(reddit.data.children.length);
+
+
+// #1
+reddit.data.children.forEach(function(allTitles) {
+	console.log(allTitles.data.title);
+});
+
+// #2
+
+var permaLinks = reddit.data.children.map(function(links) {
+		return "http://reddit.com" + links.data.permalink;
+});
+
+console.log("permaLinks:\n", permaLinks);
+// console.log("permaLinks:\n", permaLinks.length);
+
+// #3
+
+var filter = reddit.data.children.filter(function(children) {
```

Comment:
This looks better, but ```children``` as a parameter might be misleading, because that's the name of the array being iterated over. For readability, ```child``` is probably best here.